_data/template.yml

```diff
@@ -6,6 +6,7 @@ color:
   muted: 777
   mkpyellow: fed136
   mkpblue: 1f396d
+  buttonyellow: eebe1c
```

Comment:
minor, should we use mkpyellow to stay true to mkp theme? I think I also saw #888 being used, is that something we can give a name to?

Reply:
Probably should. 

I wanted to make the button a little darker for the yellow text to be more readable.